src/utils.cpp

```diff
@@ -179,6 +179,28 @@ namespace sdk {
 #endif
     }
 
+    std::string extract_tx_from_psbt(const std::string& psbt)
+    {
+#ifdef BUILD_GDK_RUST
+        return ga_rust::extract_tx_from_psbt(b2h(base64_to_bytes(psbt)));
+#else
+        (void)psbt;
+        GDK_RUNTIME_ASSERT_MSG(false, "PSBT functions not implemented");
+        return std::string();
+#endif
+    }
+
+    std::string merge_tx_in_psbt(const std::string& psbt, const std::string& tx)
+    {
+#ifdef BUILD_GDK_RUST
+        return base64_from_bytes(h2b(ga_rust::merge_tx_in_psbt(b2h(base64_to_bytes(psbt)), tx)));
```

Comment:
As above, the number of conversions/copies here makes no sense.

Reply:
commented above